boards/FSM/Src/App/states/App_FaultState.c

```diff
@@ -17,12 +18,20 @@ void App_FaultStateRunOnTick100Hz(struct StateMachine *state_machine)
 {
     App_AllStatesRunOnTick100Hz(state_machine);
 
-    // SUPRESS CERTAIN MESSAGES
-    App_CanTx_FSM_Apps_PappsMappedPedalPercentage_Set(0.0f);
+    // Suppress accelerator pedal percentages (set to 0%)
+    App_CanTx_FSM_Apps_PappsMappedPedalPercentage_Set(0);
+    App_CanTx_FSM_Apps_SappsMappedPedalPercentage_Set(0);
+
+    const bool hb_ok = !App_CanAlerts_GetFault(FSM_FAULT_MISSING_HEARTBEAT);
+    if (hb_ok)
+    {
+        App_SharedStateMachine_SetNextState(state_machine, App_GetDriveState());
+    }
```

Comment:
If heartbeat goes OK, FSM will exit fault state. Could use a review on this.

Reply:
as long as this is the only reason to go into fault state I don't see any issue with this